runtime/jvmti/jvmtiStartup.c

```diff
@@ -1062,10 +1061,56 @@ createXrunLibraries(J9JavaVM * vm)
 	return JNI_OK;
 }
 
+/**
+ * Test if an agent library was already loaded.
+ *
+ * @param vm Java VM
+ * @param library library name
+ * @param decorate a boolean indicating if the prefix/suffix to the library name is to be added
+ *
+ * @return TRUE if the agent library was loaded successfully, FALSE otherwise
+ */
 static BOOLEAN
-isAgentLibraryLoaded(J9JavaVM *vm, const char *library)
+isAgentLibraryLoaded(J9JavaVM *vm, const char *library, BOOLEAN decorate)
 {
-	return NULL != findAgentLibrary(vm, library, strlen(library));
+	BOOLEAN result = FALSE;
+
+	if (NULL == findAgentLibrary(vm, library, strlen(library))) {
```

Comment:
Suggest inverting this so the condition is close to the beginning of both branches:
```c
	if (NULL != findAgentLibrary(vm, library, strlen(library))) {
		result = TRUE;
	} else {
		/* J9PORT_SLOPEN_NO_LOAD is only supported on Linux/OSX/Win32 platforms. */
```

Reply:
Inverted